test/unit/test_cli.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2015 Scality
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Tests for `swift_scality_backend.cli`.'''
+
+import unittest
+import sys
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from StringIO import StringIO
+
+from swift_scality_backend import cli
+from swift_scality_backend.policy_configuration import StoragePolicy
+
+
+class FakeStream(object):
+    def __init__(self, module, attr):
+        self.stream = StringIO()
+        self._module = module
+        self._attr = attr
+        self._orig_attr = None
+
+    def __enter__(self):
+        self._orig_attr = getattr(self._module, self._attr)
+        setattr(self._module, self._attr, self.stream)
+
+        return self
+
+    def __exit__(self, exc, value, tb):
+        setattr(self._module, self._attr, self._orig_attr)
+
+
+class Namespace(object):
+    def __init__(self, **kwargs):
+        for (name, value) in kwargs.iteritems():
+            setattr(self, name, value)
+
+
+class TestStoragePolicyLint(unittest.TestCase):
+    def test_lint_fails_on_malformed_file(self):
+        config = 'test'
+
+        args = Namespace(
+            config=StringIO(config))
+
+        with FakeStream(sys, 'stderr') as stderr:
+            rc = cli.storage_policy_lint(args)
+
+            self.assertRegexpMatches(
```

Comment:
I think you should use utils.assertRegexpMatches here for Python 2.6 compatibility. Or probably skip these tests on Python 2.6.